assets/src/components/detourListPage.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react"
+import { DetoursTable } from "./detoursTable"
+
+export interface Detour {
+  number: number
```

Comment:
I think this needs to be a string? https://www.mbta.com/schedules/627/line

Reply:
Oh definitely, that was a careless mistake, especially given the routePill takes the id in the format of a string! And the SLs! :shakes-head: